telephony/java/android/telephony/CellInfoGsm.java

```diff
@@ -61,12 +64,20 @@ public void setCellSignalStrength(CellSignalStrengthGsm css) {
         mCellSignalStrengthGsm = css;
     }
 
+    public String getOriginalOperatorNumeric() {
```

Comment:
it sounds weird to me that you added one hidden and one not-hidden function.

Did you had to regenerate APIs/SDK?
I think we need both of those functions hidden?

Reply:
Ah you are right. I copied from the other method in that class without thinking about the Public APIs. Will clean up and update the PR.